tests/unittest_brain_pathlib.py

```diff
@@ -0,0 +1,84 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+import pytest
+
+import astroid
+from astroid import bases
+from astroid.const import PY310_PLUS
+from astroid.util import Uninferable
+
+
+def test_inference_parents() -> None:
+    """Test inference of ``pathlib.Path.parents``."""
+    name_node = astroid.extract_node(
+        """
+    from pathlib import Path
+
+    current_path = Path().resolve()
+    path_parents = current_path.parents
+    path_parents
+    """
+    )
+    inferred = name_node.inferred()
+    assert len(inferred) == 1
+    assert isinstance(inferred[0], bases.Instance)
+    assert inferred[0].qname() == "pathlib._PathParents"
+
+
+def test_inference_parents_subscript_index() -> None:
+    """Test inference of ``pathlib.Path.parents``, accessed by index."""
+    name_node = astroid.extract_node(
+        """
+    from pathlib import Path
+
+    current_path = Path().resolve()
+    parent_path = current_path.parents[2]
+    parent_path
+    """
+    )
+    inferred = name_node.inferred()
+    assert len(inferred) == 1
+    if PY310_PLUS:
+        assert isinstance(inferred[0], bases.Instance)
+        assert inferred[0].qname() == "pathlib.Path"
+    else:
+        assert inferred[0] is Uninferable
```

Comment:
`tuple` should not be correct; when passed an integral index (as opposed to a slice), you get a single path.

Reply:
No, but since this PR changes the behaviour here shouldn't we try and infer the correct value? Or do you think that is outside of the scope of this PR?

Reply:
@DanielNoord Sorry for the delay in taking a look at this. I've removed the `if PY310_PLUS` condition, and the brain tip seems to be testing fine for older Python versions.